src/pykeen/nn/init.py

```diff
@@ -244,37 +245,32 @@ class LabelBasedInitializer(PretrainedInitializer):
             embedding_dim=768,  # for BERT base
             entity_initializer=LabelBasedInitializer.from_triples_factory(
                 triples_factory=dataset.training,
+                encoder="transformer",
             ),
         )
     """
 
     def __init__(
         self,
         labels: Sequence[str],
-        pretrained_model_name_or_path: str = "bert-base-cased",
+        encoder: HintOrType[TextEncoder] = None,
+        encoder_kwargs: OptionalKwargs = None,
```

Comment:
my biggest problem with such generalizations is how do I know which arguments are supported in `encoder_kwargs`? Are they generally pykeen-specific or I have to look in another library (transformers in this case)? If so, where to look them for? I doubt users know by heart their encoder kwargs 🤔

Reply:
they depend on the `TextEncoder` you want to use :sweat_smile: 

In the code, you can follow `TextEncoder` to get to subclasses of `TextEncoder` to choose from. The default can be seen from `text_encoder_resolver`.

In the generated documentation, the annotation `TextEncoder` is clickable and leads you to the `TextEncoder` class. I am not sure whether the subclasses are linked automatically there though.

The alternative would be to copy all keyword arguments and then use the "appropriate" ones; however, this will lead to opaquely ignoring some of the keyword arguments :confused:

Reply:
> it would maybe be worth to add one doc page on this

yes, raise my both hands for it 🙌